modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbMvPartitionStorage.java

```diff
@@ -178,11 +180,29 @@ public class RocksDbMvPartitionStorage implements MvPartitionStorage {
     /** Key to store applied index value in meta. */
     private final byte[] lastAppliedIndexKey;
 
+    /** Key to store applied term value in meta. */
+    private final byte[] lastAppliedTermKey;
```

Comment:
We could, but should we? This makes a code arguably more difficult to understand.

In `TxStateRocksDbStorage` this is what I did: I store them under the same key, but there we don't use a separate meta CF, so we only have 1 key that can be used in a 'not too ugly way'.

Reply:
Maybe I'm preemptively thinking about performance. One insertion is better than two

Reply:
It is an insertion happening in memory inside a batch. It seems that even if we change it to one insert it will be really difficult to see any difference, even under load (where we'll probably be capped by FS performance, not by this difference).

Anyway, I don't have a strong opinion on this. If you believe we should change this, please let me know.